sphinx/source/docs/user_guide/jupyter.rst

```diff
@@ -11,24 +11,24 @@ Working in Notebooks
 `Jupyter`_ notebooks are computable documents often used for exploratory work,
 data analysis, teaching, and demonstration. A notebook is a series of *input
 cells* that can be individually executed to display their output immediately
-after the cell. In addition to  *Classic Notebooks* there are also notebooks for
+after the cell. In addition to  *Classic Notebooks*, there are also notebooks for
 the newer *JupyterLab* project. Bokeh can embed both standalone and Bokeh server
 content with either.
 
 .. _Jupyter:  https://jupyter.org
 
 .. _userguide_jupyter_notebook_inline_plots:
 
-Standalone output
+Standalone Output
 ~~~~~~~~~~~~~~~~~
 
 Standalone Bokeh content (i.e. that does not use a Bokeh server) can be embedded
-directly in classic Jupyter notebooks as well as in JupyterLab.
+directly in classic Jupyter Notebooks as well as in JupyterLab.
```

Comment:
I'm unclear on the right choice here. Jupyter Notebook (singular) is definitely a proper noun. But at least in my mind, "Jupyter notebooks" is an adjective and a plain noun

Reply:
c.f https://jupyter-notebook.readthedocs.io/en/stable/notebook.html where every non-title use of "notebooks" is lowercased (but also not prefaced with Jupyter) Maybe just "notebooks" is the ideal, since this is in the "Jupyter" chapter and that is implicit

Reply:
Yes, my impression was that "Jupyter Notebook" is a proper name and should thus be capitalized. Whenever it is just 'notebook' on its own, it should not be capitalized. Unfortunately, jupiter.org seems to handle this inconsistently as well (e.g., in https://jupyter-notebook.readthedocs.io/en/stable/contributing.html in the documentation -- they even use both versions on their homepage, https://jupyter.org/).

I guess we could go either way then, as long as we are consistent. I think that a lot of uppercase words can sometimes break up the flow of reading. So, the more I think about it, the more I favor the lowercase version of "Jupyter notebook".

However, I think that we should keep most occurrences of "Jupyter notebook" in jupyter.rst, mainly to prevent any possible confusion with notebooks in JupyterLab and JupyterHub (both of which are also talked about in this chapter).

How about the following: 

- We always use a lowercase "Jupyter notebook" and I'll make sure that we are consistent throughout the User Guide.
- In Jupiter.rst, I'll replace some of the occurrences of "Jupyter notebook" with "notebook", making sure not to create any confusion with notebooks in JupyterLab and JupyterHub.

What do you think?

BTW, this would also be something to include in a prospective collection of style guidelines for future documentation writers and editors! 😃

Reply:
That plan sounds great 👍